formats/keep/convert-json-to-md.ts

```diff
@@ -0,0 +1,33 @@
+import { sanitizeTags } from '../../util';
+import { KeepJson } from "./models/keep-json";
+
+/**
+ * Reads a Google Keep JSON file and returns a markdown string.
+ */
+export function convertJsonToMd(jsonContent: KeepJson): string {
+    let mdContent = '';
```

Comment:
Let's make this into an array and use `join('\n')` at the end to avoid multiple string concatenations.

Reply:
Fixed & pushed